src/components/Callout/styled.js

```diff
@@ -0,0 +1,12 @@
+import styled from "styled-components";
+import { PINK, BLACK } from "../../styles/styles";
+
+export const Container = styled.div`
```

Comment:
in the deploy preview the heading and button seem to wrap on tablet and mobile:
<img width="413" alt="Screen Shot 2022-04-13 at 10 21 19 AM" src="https://user-images.githubusercontent.com/52389456/163203191-94689991-7873-4ca5-991f-9b3179ae1127.png">